src/main/scala/io/iohk/ethereum/blockchain/data/GenesisDataLoader.scala

```diff
@@ -94,7 +94,7 @@ class GenesisDataLoader(
     val initalRootHash = MerklePatriciaTrie.EmptyRootHash
 
     val stateMptRootHash = genesisData.alloc.zipWithIndex.foldLeft(initalRootHash) { case (rootHash, (((address, AllocAccount(balance)), idx))) =>
-      val ephemNodeStorage =  PruningMode.nodesKeyValueStorage(pruning.ArchivePruning, nodeStorage)(Some(idx - genesisData.alloc.size))
+      val ephemNodeStorage = PruningMode.nodesKeyValueStorage(pruning.ArchivePruning, nodeStorage, withSnapshotsSave = true)(Some(idx - genesisData.alloc.size))
```

Comment:
shouldn't this one be false as the one that  @ntallar mentions in https://github.com/input-output-hk/mantis/pull/433/files#r178815627

Reply:
good point, we probably do not need snapshots to revert past genesis block 👍